src/Testing/LevelsTestCase.php

```diff
@@ -39,7 +39,7 @@ public function testLevels(
 
 		foreach (range(0, 7) as $level) {
 			unset($outputLines);
-			exec(sprintf('php %s analyse --no-progress --error-format=prettyJson --level=%d %s --autoload-file %s %s', $command, $level, $configPath !== null ? '--configuration ' . escapeshellarg($configPath) : '', escapeshellarg($file), escapeshellarg($file)), $outputLines);
+			exec(sprintf('%s %s analyse --no-progress --error-format=prettyJson --level=%d %s --autoload-file %s %s', PHP_BINARY, $command, $level, $configPath !== null ? '--configuration ' . escapeshellarg($configPath) : '', escapeshellarg($file), escapeshellarg($file)), $outputLines);
```

Comment:
It should be escaped with `escape_shell_arg()`, right?

Reply:
Although unlikely to be needed, yes:
```
# "/usr/bin/p'hp" -r 'passthru(PHP_BINARY . " -v");'
sh: 1: Syntax error: Unterminated quoted string

# "/usr/bin/p'hp" -r 'passthru(escapeshellarg(PHP_BINARY) . " -v");'
PHP 7.3.5-1+ubuntu19.04.1+deb.sury.org+1 (cli) (built: May  3 2019 10:01:08) ( NTS )
```